converters/google/convert_test.go

```diff
@@ -480,3 +480,97 @@ func TestTimestampUnmarshalJSON(t *testing.T) {
 	}
 	assert.EqualValues(t, ts, expected)
 }
+
+func TestAssetParent(t *testing.T) {
+	tests := []struct {
+		name      string
+		ancestors []string
+		tfData    resources.TerraformResourceData
+		cai       *resources.Asset
+		want      string
+		wantErr   bool
+	}{
+		{
+			name:      "project",
+			ancestors: []string{"projects/123", "folders/456", "organizations/789"},
+			cai:       &resources.Asset{Type: "cloudresourcemanager.googleapis.com/Project"},
+			want:      "//cloudresourcemanager.googleapis.com/folders/456",
+		},
+		{
+			name:      "project with no other ancestors",
+			ancestors: []string{"projects/123"},
+			cai:       &resources.Asset{Type: "cloudresourcemanager.googleapis.com/Project"},
+			want:      "//cloudresourcemanager.googleapis.com/projects/123",
+		},
+		{
+			name:      "folder",
```

Comment:
yes, I think? this would be if the folder is being created with Terraform, and the parent isn't yet known

Reply:
- folder creation is in `google_folder` resource, and currently there is no converter to support that.
- for folder creation, [no folder_id](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_folder) is in the context, and parent, display_name are required fields in terraform data. In future, if we support `google_folder` resource, more logics need to be adjust. eg. 
  - In this PR, the ancestry manager will use API to fetch `folder` type asset parent. For folder creation, this should not be done. 
- seems folder should always have a parent, for creation, this is defined in parent, for existing folder, it can be fetched from API.

Reply:
Ah - yeah, this is irrelevant since there's no google_folder resource.